tests/helpers/buildNextApp.js

```diff
@@ -19,9 +19,6 @@ const { NEXT_VERSION } = require("./nextVersion");
 // run next-on-netlify.
 
 class NextAppBuilder {
-  // Name of the app to build. This determines the build path.
-  __appName = null;
```

Comment:
Using instance fields is not supported in Node 10.

`__appName` is used below in `path.join()` which expects a specific value to have been configured with `forTest()`. So this can be safely removed.

Reply:
things i wouldve probably never found: this